eng/pipelines/templates/steps/build-test.yml

```diff
@@ -24,8 +24,6 @@ steps:
     parameters:
       versionSpec: '${{ parameters.PythonVersion }}'
 
-  - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
```

Comment:
I'm OK with us disabling this temporarily to onboard the pool but please file an issue to track enabling some way to validate the machines are as we expect. We should either have something that runs in the pipeline or have some external pipeline that can run on the different pools to assert they are giving use the correct OS images.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/1313